added steps (#2139)
SumoLogic/sumologic-documentation: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md

## Before you start

> If this is the first time you've deployed the AWS Observability Solution, read the [Before You Deploy](https://help.sumologic.com/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information.

Reviewer: ```suggestion
If this is the first time you've deployed the AWS Observability solution, read the [Before You Deploy](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information.
```

> If this is the first time you've deployed the AWS Observability solution, read the [Before You Deploy](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information. And, complete the prerequisites for StackSets as described in the [AWS documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).

Reviewer: ```suggestion
* If this is the first time you've deployed our AWS Observability solution, read the [Before You Deploy](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information.
* Complete the prerequisites for StackSets as described in the [AWS documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).
```

## Step 1: Open the CloudFormation template

> 1. Sign on to the AWS Management console.

Reviewer: ```suggestion
1. Sign in to the AWS Management console.
```

1. Choose an option:
   * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml)

>    * Click the AWS Observability Solution (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template. If you would like to download or inspect this or other versions of this template, please visit [Change Log](../changelog.md).

Reviewer: We don't use 'please'

```suggestion
   * Click the AWS Observability solution (S3 Link for CloudFormation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template. If you would like to download or inspect this or other versions of this template, see [Change Log](../changelog.md).
```

    :::note
    If you would like to download or inspect this or other versions of this template, see the [Changelog](../changelog.md).
    :::
1. Select the AWS Region where you want to deploy the AWS CloudFormation template.
    :::warning

>     This step is critical: if you do not select the correct region, you will deploy the solution in the wrong region.

Reviewer: ```suggestion
    This step is critical. If you do not select the correct region, you will deploy the solution in the wrong region.
```

1. Sign on to the AWS Management console.

> 1. Choose an option:

Reviewer 2: Can we write why the user needs to choose the option?

Author: updated

1. Enable ELB Classic Access logging as **None** and Create Sumo Logic ELB Classic Logs Source as **No**. <br/>  ![Multiaccount6.png](/img/observability/Multiaccount6.png)
1. Location where you want the App to be Installed as **PersonalFolder**. And for **Do you want to share App with whole organization**, set as **True**.<br/>  ![Multiaccount7.png](/img/observability/Multiaccount7.png)

> ## Step 5: Determine Account Aliases

Reviewer 2: ```suggestion
## Step 5: Determine account aliases
```

    in your AWS account.
1. Click **Create StackSet**.  
    ![CloudFormation_Stackset 1.png](/img/observability/ClodFormation_Stackset_1.png)

> 1. Paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option and click **Next**. If you would like to download or inspect this or other versions of this template, please visit the [change log](../changelog.md) page.<br/>  ![multi-create-stack.png](/img/observability/multi-create-stack.png)

Reviewer: ```suggestion
1. Paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option and click **Next**. If you'd like to download or inspect this or other versions of this template, see the [Changelog](../changelog.md) page.<br/>  ![multi-create-stack.png](/img/observability/multi-create-stack.png)
```
